src/iio_wrapper.cpp

```diff
@@ -41,7 +41,10 @@ IIOWrapper::IIOWrapper()
 {
     if(m_local_context == nullptr)
     {
-        m_local_context = iio_create_local_context();
+        std::string uri = "ip:127.0.0.1"; // TODO: pass uri from cmd line
+
+        m_local_context = iio_create_context_from_uri(uri.c_str());
+        //m_local_context = iio_create_local_context();
```

Comment:
you keep changing these lines through the commit... change it just once in the final form